### ISSUE TITLE
Total Rewards Link routes to "not found"

### ISSUE THREAD

**Issue by jonmhong** [jlevy/og-equity-compensation#246]
Under the "Compensation and Equity" header, the "total rewards" link is broken.

It routes to https://www.worldatwork.org/total-rewards-model/
But it should instead route to https://www.worldatwork.org/total-rewards/

<img width="447" alt="Screen Shot 2022-03-17 at 3 54 41 PM" src="https://user-images.githubusercontent.com/13195717/158884978-2faf87eb-d359-4679-b56b-d2d6bbdbded4.png">

**Comment by griffeth** [comment]
I don't think this is an issue anymore...I clicked on the existing link (https://www.worldatwork.org/total-rewards-model/) and the suggested link (https://www.worldatwork.org/total-rewards/) and both landed at https://worldatwork.org/

I don't know if that was the originally intended landing spot but the links above are being redirected now instead of not finding the page.
